### PR TITLE
Update/types

### PR DESCRIPTION
**Pull request by patrikliska**
In @types/react 18 there is error https://github.com/redwoodjs/redwood/issues/5104#issuecomment-1094347397 
@types/react needs to be updated in this library for proper behavior

### CODE REVIEW

**Comment by davidslaby** [comment]
@sghall Hi, could you check please, merge and publish new version of package? Otherwise, it's incompatible with React 18 types.

**Review by davidslaby — APPROVED** [review]

File: src/Slider/Slider.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ interface RCSComponent {
   };
 }
 
-const isRCSComponent = (item: React.ReactNode): item is RCSComponent => {
+const isRCSComponent = (item: React.ReactNode) => {
```
Comment (patrikliska) [inline src/Slider/Slider.tsx:39]: This is automatically typed

**Comment by lachlanjc** [comment]
Also having issues with this, would love to see the new version published!

**Comment by bentron2000** [comment]
Ditto!

**Comment by sghall** [comment]
Hey 👋  I will take a look at this over the weekend.  Apologies for the delay.

**Comment by andreyvital** [comment]
@sghall any change of it getting merged? 🤔

**Pull request merged**